java/base-client/src/main/java/no/bankaxept/epayment/client/base/accesstoken/ScheduledAccessTokenPublisher.java

```diff
@@ -116,4 +114,74 @@ public void subscribe(Flow.Subscriber<? super String> subscriber) {
             }
         }
     }
+
+    public static class Builder {
+        private String uri;
+        private HttpClient httpClient;
+        private ScheduledExecutorService scheduler;
+        private Clock clock = Clock.systemDefaultZone();
+
+        private final Map<String, List<String>> headers = new HashMap<>(Map.of("Content-type", List.of("application/x-www-form-urlencoded")));
```

Comment:
I put this value in here now, since it should always be set now. also avoids inserting it multiple times if you build() again, though it shouldn't matter too much